ts/store/reducers/entities/messages/allPaginated.ts

```diff
@@ -212,6 +212,11 @@ export const allMessagesSelector = createSelector(
     )
 );
 
+export const getById = createSelector(
+  [allMessagesSelector, (_: GlobalState, messageId: string) => messageId],
+  (page, messageId) => page.find(_ => _.id === messageId)
```

Comment:
```suggestion
  (page, messageId): UIMessage | undefined => page.find(_ => _.id === messageId)
```
nothing important, just to improve readability